app/routes.js

```diff
@@ -60,7 +61,10 @@ var BreadcrumbMaker = require('../lib/js/breadcrumb_maker.js');
         res.render('content', { content: content, breadcrumb: breadcrumb, taxons: taxons, whitehall: whitehall, homepage_url: '/'});
       },
       function (e) {
-        res.render('content', {content: 'Page not found', homepage_url: '/'});
+        readFile(error404Page).then(function(data) {
+          content = data.toString();
```

Comment:
Rather than inject the contents of the 404 template into the content template, it would be better to just render the 404 template directly here.

i.e. something like
```
res.status(404).render('errors/404.html')
```